phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexRebuildTaskIT.java

```diff
@@ -178,11 +178,15 @@ public static void waitForTaskState(Connection conn, PTable.TaskType taskType, S
         String taskData = "";
         do {
             Thread.sleep(2000);
-            ResultSet rs = conn.createStatement().executeQuery("SELECT * " +
+            String stmt = "SELECT * " +
                     " FROM " + PhoenixDatabaseMetaData.SYSTEM_TASK_NAME +
-                    " WHERE " + PhoenixDatabaseMetaData.TABLE_NAME + "='" + expectedTableName + "' AND " +
```

Comment:
Let's keep the table name in the query. When the task table is used for other use cases, there might be other entries from some other tables in the task table even during this IT